exonum/src/blockchain/mod.rs

```diff
@@ -98,6 +100,14 @@ impl Blockchain {
         }
     }
 
+    /// Recreates the blockchain to reuse with a sandbox.
```

Comment:
And what will happen with the `Mount`s which were created by `mount_*_api`. It seems that they will continue to use an old `api_sender` instance.
If this method is needs only for sandbox testing it should be marked by `#[doc(hidden)]` attribute.